rest/route/annotations.go

```diff
@@ -258,7 +258,7 @@ func (h *bulkPatchAnnotationHandler) Run(ctx context.Context) gimlet.Responder {
 // PUT /rest/v2/tasks/{task_id}/annotation
 
 // Parsing logic for task annotation put and patch routes.
-func annotationByTaskPutOrPatchParser(ctx context.Context, r *http.Request) (string, *restModel.APITaskAnnotation, error) {
+func annotationByTaskPutOrPatchParser(r *http.Request) (string, *restModel.APITaskAnnotation, error) {
```

Comment:
Threading context here allows us to capture this as part of one trace.  I think we should keep it. Jonathan is planning on doing some work to thread context for allot of our requests and db calls.